pkg/cluster/cluster.go

```diff
@@ -260,8 +272,16 @@ func prepareBaseRootfs(rootfsDir string, clusterSettings *ClusterSettings) error
 	if err := fs.CreateFileFromString(path.Join(rootfsDir, "/etc/resolv.conf"), "nameserver 8.8.8.8"); err != nil {
 		return err
 	}
+
 	if clusterSettings.ContainerRuntime == "rkt" {
-		if err := fs.CreateFileFromString(path.Join(rootfsDir, "/etc/systemd/system/rktlet.service"), RktletSystemdUnit); err != nil {
+		buf, err := ExecuteTemplate(RktletSystemdUnitTmpl, clusterSettings)
+		if err != nil {
+			log.Printf("*************ERROR**********", err)
```

Comment:
Please remove this print statement and the similar one below and return only an error. We aim to let the caller handle error reporting.